packages/backend/models/Note.js

```diff
@@ -9,6 +10,7 @@ let noteSchema = new Schema({
     required: true,
   },
   note: { type: String, required: true },
+  date: { type: Date, default: m(Date.now()).toISOString() },
```

Comment:
All you need is this:
```suggestion
  date: { type: Date, default: m().toISOString() },
```